src/Factory/FormLayoutFactory.php

```
@@ -395,6 +405,7 @@ public static function createFromFieldDtos(?FieldCollection $fieldDtos): FieldLa
                 $tabs[$fieldDto->getUniqueId()] = $fieldDto;
             } else {
                 if ($hasTabs) {
+                    /** @phpstan-ignore-next-line offsetAccess.nonOffsetAccessible */
```

The method is deprecated, let's just ignore the error